src/engine/helper/pka_helper.c

```diff
@@ -181,6 +181,103 @@ static void free_operand(pka_operand_t *operand)
     free(operand);
 }
 
+/* ECC related helpers */
+
+static void make_ecc_operand(PKA_ULONG     *bn_buf_ptr,
+                             uint32_t       buf_len,
+                             uint32_t       buf_max_len,
+                             uint8_t        big_endian,
+                             pka_operand_t *operand)
+{
+    if (operand == NULL)
+        return;
+
+    if (!bn_buf_ptr || (buf_max_len == 0))
+        return;
+
+    memset(operand, 0, sizeof(pka_operand_t));
+    operand->big_endian = big_endian;
+
+    // Now init the operand buffer
+    operand->buf_ptr    = malloc(buf_max_len);
+    operand->buf_len    = buf_max_len;
+    memset(operand->buf_ptr, 0, buf_max_len);
+
+    // Now fill the operand buf.
+    operand_byte_copy(operand, (uint8_t *) bn_buf_ptr, buf_len);
+
+}
+static void bignum_to_ecc_operand(pka_bignum_t  *bignum,
+                                  pka_operand_t *operand)
+{
+    if (operand == NULL)
+        return;
+
+    uint32_t byte_len, byte_max_len;
```

Comment:
Must be at the top. Please do not mix code and variable declaration.

Reply:
Ack, Thanks.